src/components/ebay-area-chart/component.js

```diff
@@ -0,0 +1,331 @@
+import Highcharts from 'highcharts';
+import accessibility from 'highcharts/modules/accessibility';
+import patternFill from 'highcharts/modules/pattern-fill';
+import {
+    backgroundColor,
+    gridColor,
+    labelsColor,
+    legendColor,
+    legendInactiveColor,
+    legendHoverColor,
+    tooltipBackgroundColor,
+    tooltipShadows,
+    setSeriesColors,
+} from '../../common/charts/colors';
+
+import { ebayLegend } from '../../common/charts/legend';
+
+if (typeof Highcharts === 'object') {
+    // enable highcharts accessibility with wrapper function
+    accessibility(Highcharts);
+    // patternFill highcharts wrapper function enables rendering patterns instead of just solid colors
+    patternFill(Highcharts);
+    // add custom legend wrapper function
+    ebayLegend(Highcharts);
+}
+const pointSize = 1.5;
+
+export default class {
+    onCreate() {
+        this.chartRef = null;
+        this.state = {
+            containerId: `ebay-bar-chart-${this.id}`, // set unique ID for html container element that highcharts will use
+        };
+    }
+    onMount() {
```

Comment:
Can we have a `setup` method, like we have for video, or other components (`_setupMakeup` etc)
`onMount` just calls those setup methods (we can have a `_setupEvents` and a `_setupCharts` one)